docs/source/index.rst

```diff
@@ -11,4 +11,4 @@ HPAT documentation
    DevelopersGuide
    Modules
 
-To download HPAT documentation PDF click here :download:`pdf <../_build/latex/HPAT.pdf>`
\ No newline at end of file
+To download HPAT documentation PDF click here :download:`pdf <../_build/latex/HPAT.pdf>`
```

Comment:
it will not work on Linux (we have 'build' directory there instead '_build'

Reply:
`_build` is the folder that sphinx creates (not the one from setuptools). There is no difference between OSes.

Reply:
@fschlimb In this case this directory should be under `build`. We don't need several temporary directories in top project directory.

Reply:
We don't need to use the pdf in _build directory. We can copy the pdf in other directory and use that link. But it should be mentioned explicitly that if someone has added changes to documentation and built it then he/she needs to build the pdf and update it in the location mentioned in the link.

Reply:
Hm, the building sphinx docs is a separate process, it typically happens in the `docs` directory, not top level. It is probably possible to change default behavior. I don't think it's important, though.

In any case, in this extra process, sphinx creates a directory (e.g. `_build`) where it puts its output. The `html` directory in there is then what gets 'manually' copied to wherever  the webserver expects to see the files. I would expect (but I don't know, so you should check) that at build time sphinx will copy files referenced in the output dir to the html dir. Otherwise our 'manual' copy procedure needs to include the (path to) the pdf so that the webserver finds it.